pages/api/profiles.js

```diff
@@ -1,5 +1,24 @@
-import data from "../../.cache/data.json";
+import fs from "fs";
+import path from "path";
 
-export default function handler(req, res) {
-  res.status(200).json(data)
+export default async function handler(req, res) {
+  const relativeDir = path.resolve("./public", "profiles");
+  const files = fs.readdirSync(relativeDir);
+
+  const profiles = await Promise.all(
+    files.map(async (file) => {
+      const response = await fetch(
+        `http://localhost:3000${path.join("/", "profiles", file)}`
```

Comment:
We should maybe think of a solution of not hardcoding the URL although it needs to be a absolute one.
(Afaik, in index.js the Request-URL is also hardcoded.)

Reply:
I think if a request url the `join` is not required, only if it is a disk path

I wonder if we can use a relative url here? `/profiles/...`